[AIRFLOW-2493] Mark template_fields of all Operators in the API document as "templated"

imroc opened the pull request:
### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2493


### Description
- [x] Here are some details about my PR:
Make all the "template_fields" (jinjia template) of all Operators marked as "templated" in the API document.

### Tests
- [x] My PR does not need testing for this extremely good reason:
just update document, no need for tests.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [x] no new functionality, just update document.

kaxil commented on airflow/operators/hive_to_mysql.py:
```diff
@@ -30,23 +30,23 @@ class HiveToMySqlTransfer(BaseOperator):
-        import, typically used to move data from staging to production
-        and issue cleanup commands.
+        import (templated), typically used to move data from staging to
+        production and issue cleanup commands.
```
I think you should have `(templated)` at the end of the line here as well instead of before comma `,`

kaxil commented on airflow/operators/presto_to_mysql.py:
```diff
@@ -28,19 +28,19 @@ class PrestoToMySqlTransfer(BaseOperator):
-        import, typically use to truncate of delete in place of the data
-        coming in, allowing the task to be idempotent (running the task
-        twice won't double load data)
+        import (templated), typically use to truncate of delete in place
```
same here. Please add `(templated)` at the end of the line instead of before comma `,`

kaxil commented on airflow/operators/slack_operator.py:
```diff
@@ -87,15 +87,17 @@ class SlackAPIPostOperator(SlackAPIOperator):
     :param icon_url: url to icon used for this message
     :type icon_url: string
-    :param attachments: extra formatting details - see https://api.slack.com/docs/attachments
+    :param attachments: extra formatting details
```
Please add `(templated)` at the end of the description not after reference link

imroc committed:
[AIRFLOW-2493] Mark template_fields of all Operators in the API document as "templated"

Make all the "template_fields" (jinjia template) of all Operators marked as "templated" in the API document.

kaxil reviewed (approved):
LGTM.

Thank You for your contribution.
